dash/App/Accounts/Add/AddAddress/index.js

```diff
@@ -127,8 +128,16 @@ class AddAddress extends React.Component {
                   <div className='addAccountItemOptionSubmit' onClick={() => this.create()}>Create</div>
                 </div>
                 <div className='addAccountItemOptionSetupFrame'>
-                  <div className='addAccountItemOptionTitle'>{this.state.status}</div>
-                  {this.state.error ? <div className='addAccountItemOptionSubmit' onClick={() => this.restart()}>try again</div> : null}
+                  {error ? 
+                  <>
+                    <div className='addAccountItemOptionTitle'>{this.state.status }</div>
+                    <div className='addAccountItemOptionSubmit' onClick={() => this.restart()}>try again</div>
+                  </>
+                  :
+                  <>
+                   <div className='addAccountItemOptionTitle'>{'account added successfully'}</div>
+                   <div className='addAccountItemOptionSubmit' onClick={() => link.send('tray:action', 'navBack', 'dash', 2)}>back</div>
```

Comment:
thought we decided we were gonna use the nav-specific actions here?

Reply:
`link.send('nav:back', 'dash', 2)`

Reply:
this works for me, will upload the changes so you can try

Reply:
LGTM!